tendrl/commons/utils/central_store/fields.py

```diff
@@ -244,7 +244,7 @@ def _set_value(self, value):
 
         # Force casting if we were given a caster.
         if self._caster:
-            for x in value.keys():
+            for x in list(value.keys()):
```

Comment:
I am seeing these changes in a few places, it is not necessary to convert this list by default it will give the result in list only. 

But not sure please check :)

Reply:
no need list(value.keys()) or list((value.values()) or len(list((value.keys())) no need list conversion if i am right

Reply:
Done